src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -567,9 +568,10 @@ export const Deposit: React.FC<DepositProps> = ({
     browserHistory.goBack()
   }, [browserHistory])
 
-  const validateCryptoAmount = useCallback(
+  const _validateCryptoAmount = useCallback(
     async (value: string) => {
-      if (!accountId) return
+      if (!accountId || !contextDispatch) return
```

Comment:
no need for this check here and remove from callback dependency array
```suggestion
      if (!accountId) return
```

Reply:
Removed as part of rebase both here and in `<Withdraw />`